jiant/bert/utils.py

```diff
@@ -44,6 +44,12 @@ def __init__(self, args, cache_dir=None):
             args.input_module, cache_dir=cache_dir
         )
         self.embeddings_mode = args.bert_embeddings_mode
+        self.num_layers = self.model.config.num_hidden_layers
+        if args.bert_max_layer >= 0:
+            self.max_layer = args.bert_max_layer
+        else:
+            self.max_layer = self.num_layers
+        assert self.max_layer <= self.num_layers
```

Comment:
Move this inside the first conditional clause?

Reply:
It's a little safer to enforce globally?

Reply:
Up to you, but seems more confusing then helpful. The pattern is `x = y; assert x >= y`.